src/constants.ts

```diff
@@ -45,6 +45,8 @@ export const CONSTANTS = {
       "error.debuggingSessionInProgress",
       "[ERROR] A debugging session is currently in progress, please stop it before running your code. \n"
     ),
+    DEPENDENCY_DOWNLOAD_ERROR:
+      "Package downloads failed. Some functionality may not work. Try restarting the simulator or review the installation docs.",
```

Comment:
Does "functionalities" sound better than "functionality"?

Reply:
yup! I'll fix that.